config.json

```diff
@@ -4,68 +4,98 @@
-      }, {
-        "destination": "variables-size.css",
-        "format": "css/variables",
-        "filter": "isCategorySize"
-      }]
+      "files": [
+        {
+          "destination": "variables-color.css",
+          "format": "css/variables",
+          "filter": "isCategoryColor"
+        },
+        {
+          "destination": "variables-size.css",
+          "format": "css/variables",
+          "filter": "isCategorySize"
+        }
+      ]
     },
     "cssUtilities": {
       "buildPath": "build/utilities/",
       "transformGroup": "css",
-      "files": [{
-        "destination": "utilities-size.css",
-        "format": "utilityClass",
-        "filter": "isCategorySize"
-      }, {
-        "destination": "utilities-color.css",
-        "format": "utilityClass",
-        "filter": "isCategoryColor"
-      }]
+      "files": [
+        {
+          "destination": "utilities-size.css",
+          "format": "utilityClass",
+          "filter": "isCategorySize"
+        },
+        {
+          "destination": "utilities-color.css",
+          "format": "utilityClass",
+          "filter": "isCategoryColor"
+        }
+      ]
     },
     "sassUtilities": {
       "buildPath": "build/utilities/",
       "transformGroup": "scss",
-      "files": [{
-        "destination": "utilities-size.scss",
-        "format": "utilityClass",
-        "filter": "isCategorySize"
-      }, {
-        "destination": "utilities-color.scss",
-        "format": "utilityClass",
-        "filter": "isCategoryColor"
-      }]
+      "files": [
+        {
+          "destination": "utilities-size.scss",
+          "format": "utilityClass",
+          "filter": "isCategorySize"
+        },
+        {
+          "destination": "utilities-color.scss",
+          "format": "utilityClass",
+          "filter": "isCategoryColor"
+        }
+      ]
     },
     "js": {
       "transformGroup": "js",
       "buildPath": "build/js/",
-      "files": [{
-        "destination": "variables-color.js",
-        "format": "javascript/module",
-        "filter": "isCategoryColor"
-      }, {
-        "destination": "variables-size.js",
-        "format": "javascript/module",
-        "filter": "isCategorySize"
-      }]
+      "files": [
+        {
+          "destination": "variables-color.js",
+          "format": "javascript/module",
+          "filter": "isCategoryColor"
+        },
+        {
+          "destination": "variables-size.js",
+          "format": "javascript/module",
+          "filter": "isCategorySize"
+        }
+      ]
+    },
+    "web": {
+      "transformGroup": "js",
+      "buildPath": "build/json/",
+      "files": [
+        {
+          "destination": "variables-color.json",
+          "format": "json",
+          "filter": "isCategoryColor"
+        },
+        {
+          "destination": "variables-size.json",
+          "format": "json",
+          "filter": "isCategorySize"
+        }
+      ]
     }
```

Comment:
how does targeting work for this? how does it know to target `variables-color.js` and `variables-size.js`? is it passed on the destination file name?

Reply:
It's not targeting the JS files, it's pulling from the source files that are already set up in this repo in the properties folder! With this setup we'll be able to easily add other properties as they get added too.

You can check out build.js and the docs for style-dictionary for more info, I really only know enough to add the JSON files!

Reply:
@ericyliu the build file has filters in it that we then use in the config to write specific variables to specific folders/files